### ISSUE TITLE
Add a compact link to homepage on right navigation bar

### ISSUE THREAD

**Issue by kamalx** [punchagan/er#14]
All the navigation and links on the site are in the RHS of the screen on desktops except one - the link to homepage. This requires a long and tedious mouse travel, and then point and shoot on the home link on top left of the site on desktops. It becomes very tedious for a reader who would spend long enough time on the site. This is not a problem for those who come to read one post and leave.

I recommend a tiny Home icon linking to homepage on the top nav on RHS.

**Comment by punchagan** [comment]
I think the navbar and header could be cleaned up a little. The left title seems a little lonely, and so does the lone link on the RHS.

**Comment by kamalx** [comment]
> I think the navbar and header could be cleaned up a little. The left title seems a little lonely, and so does the lone link on the RHS.

Agreed. We'll work on revamping this next, now that #17 - search feature is sorted and its trigger button can be further refined as well.
